app/commands.py

```diff
@@ -505,7 +505,7 @@ def migrate_data_to_ft_notification_status(start_date, end_date, notification_ty
 
     start_date = start_date.date()
     end_date = end_date.date()
-    for day_diff in range((end_date - start_date).days):
+    for day_diff in range((end_date - start_date).days + 1):
```

Comment:
Is there a test you can add/change to confirm this change to an inclusive end date has worked?

Reply:
sadly not :(